tests/markdown/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -996,13 +997,10 @@ foo(arg1, arg2, arg3, arg4);
 It fits in a single line so it's going to stay as is. However, we've all run
 into this situation:
 
+<!-- prettier-ignore -->
+
```

Comment:
Is this blank line intentional?

Reply:
It always prints additional line break for continuous nodes that their types are different, should we add a special case for `prettier-ignore`?

Reply:
Yes, I think so. It will probably be reported as a bug.